p4-16/spec/P4-16-spec.mdk

```diff
@@ -6583,6 +6583,31 @@ table is an abstract finite map that is given a key and produces as a
 result either an action or a "miss" indication, as described in
 Section [#sec-mau-semantics].
 
+The expected meaning of these values is as follows:
+
+- an `exact` match kind on a key field means that the value of the
+  field in the table specifies exactly all the bits for the key field.
+  This is applicable for key fields of types `bit<>`, `int<>`,
+  serializable enums, and structs or headers whose fields have such
+  types.
```

Comment:
Would it be reasonable to generalize this to "all types for which equality is defined" ?

Reply:
Does the control plane support such types for keys? 
Perhaps the control-plane is the scope of a separate document, though, and this spec should be very liberal.